profiles/common/modules/custom/multisite_create_button/multisite_create_button.module

```diff
@@ -42,7 +42,7 @@ function _multisite_create_button_contextual_links() {
   if (!empty($content)) {
     $links = array();
     foreach ($content as $content_type) {
-      $links['items'][] = l($content_type['link_title'], $base_url . '/' . $content_type['link_path']);
+      $links['items'][$content_type['link_title']] = $base_url . '/' . $content_type['link_path'];
     }
 
     $output = theme('dropdown', $links);
```

Comment:
We do not have the same logic into function between the themes. 
The theme_dropdown() in the module define the default (good practice when you declare a new theme pattern)
In the other themes, you can see the functions that override this theme_dropdown() according with the tpl.
ec_resp: https://github.com/ec-europa/ec_resp/pull/3/files#diff-13e5580ca26ea5de947e855a28fb0774R1563
ec-europa: https://github.com/ec-europa/ec-europa-theme/pull/178/files#diff-13e5580ca26ea5de947e855a28fb0774R108